test/common/http/http2/hpack_fuzz_test.cc

```diff
@@ -90,10 +96,15 @@ std::vector<nghttp2_nv> decodeHeaders(nghttp2_hd_inflater* inflater,
 }
 
 struct NvComparator {
-  inline bool operator()(const nghttp2_nv& a, const nghttp2_nv& b) {
-    absl::string_view a_str(reinterpret_cast<char*>(a.name), a.namelen);
-    absl::string_view b_str(reinterpret_cast<char*>(b.name), b.namelen);
-    return a_str.compare(b_str);
+  inline bool operator()(const nghttp2_nv& a, const nghttp2_nv& b) const {
+    absl::string_view a_name(reinterpret_cast<char*>(a.name), a.namelen);
+    absl::string_view b_name(reinterpret_cast<char*>(b.name), b.namelen);
+    if (a_name != b_name) {
+      return a_name < b_name;
+    }
+    absl::string_view a_val(reinterpret_cast<char*>(a.value), a.valuelen);
+    absl::string_view b_val(reinterpret_cast<char*>(b.value), b.valuelen);
+    return b_val > a_val;
```

Comment:
How come this isn't `return a_val < b_val`? Isn't the idea to get lexicographical ordering?

Reply:
Hmmmm. I'm not sure if I'm reading the comment correctly, but that is what I'm returning (string_view's `b_val > a_val` returns `a_val < b_val`.) Switched ordering though